src/components/Cluster/ClusterDetail/ClusterLabels/ClusterLabels.tsx

```diff
@@ -2,19 +2,25 @@ import styled from '@emotion/styled';
 import { V5ClusterLabelsProperty } from 'giantswarm';
 import PropTypes from 'prop-types';
 import React, { FC, HTMLAttributes, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  getClusterLabelsError,
+  getClusterLabelsLoading,
+} from 'selectors/clusterLabelSelectors';
+import { updateClusterLabels } from 'stores/clusterlabels/actions';
 import LabelWrapper from 'UI/ClusterLabels/LabelWrapper';
 
 import DeleteLabelButton from './DeleteLabelButton';
 import EditLabelTooltip from './EditLabelTooltip';
 
-interface IClusterLabelsProps {
+interface IClusterLabelsProps extends HTMLAttributes<HTMLDivElement> {
```

Comment:
`HTMLAttributes` is not exactly the most suitable type because those are HTML attributes, not React attributes (some attributes have different names)
```suggestion
interface IClusterLabelsProps extends React.ComponentPropsWithoutRef<'div'> {
```